serialization/murmur3/README.md

```diff
@@ -12,4 +10,184 @@ Run with:
 
 ```
 java -jar target/murmur3-0.1-SNAPSHOT.jar 
-```
\ No newline at end of file
+```
+
+# Where Are My Keys ?
+
+It’s useful to understand how, why and where Hazelcast stores your data in the grid.
+
+What happens is what you really need, but it won’t necessarily be what you first think you want.
+
+We’ll explore this in this blog by putting some number keys in a map, a common starting point and source of confusion.
+<h2>A Map</h2>
+A “map” is a data structure in Computer Science that is sometimes called a “key-value” store or a “dictionary.” Here’s one way to think of it:
+
+<img class="size-medium wp-image-754053" src="https://hazelcast.com/wp-content/uploads/2020/06/map1-400x269.jpg" alt="A Map" width="400" height="269" />
+
+Our map here is some sort of logical collection of data pairs. We’ve named it “numbers” to distinguish it from other data collections we may have. It contains some but not all numbers, and there’s no implication that these are stored in any particular order.
+
+Five data records are shown. The left side of the entry record is the key, here a number. The right side of the entry record is the value, here a string.
+
+The keys are unique. Values don’t have to be unique. Though it makes sense for this example.
+
+It's similar to looking up a word in a real dictionary. We have a word key we wish to lookup. This is used to find and return the corresponding definition.
+
+It’s also a bit like “<code>SELECT * FROM table WHERE primaryKey = $1</code>” in SQL terms.
+
+Our data record has a primary key. We wish to retrieve using the primary key. This is what a map does best.
+<h2>IMap</h2>
+<a href="https://docs.hazelcast.org/docs/4.0.1/javadoc/com/hazelcast/map/IMap.html">IMap</a> is Hazelcast’s implementation of the classic map data structure.
+
+In a Hazelcast grid, there are usually several nodes doing data storage.
+
+In this case, the <code>IMap</code> would be spread across these processes. Each process hosts a share of the map's contents.
+
+This gives the obvious benefit of scalability. With three nodes, each node hosts a third of the map. So the map's capacity is triple the capacity of a single process. If we add a fourth process, the map's capacity is now four times that of a single process. We have scaled up.
+
+However, there is also the potential for confusion if you don't understand some of the necessary implementation details.
+
+For example, with 3 processes, each has ⅓ of the map to look after but it might not mean ⅓ of the data.
```

Comment:
Maybe `with 3 processes, each has ⅓ of the key range to look after`?

Reply:
As above, a cut&paste from https://hazelcast.com/blog/where-are-my-keys/, don't want to amend to keep the two matching